src/webhook/mutation/pod_mutator/init_container.go

```diff
@@ -18,7 +18,7 @@ func createInstallInitContainerBase(webhookImage, clusterID string, pod *corev1.
 		ImagePullPolicy: corev1.PullIfNotPresent,
 		Args:            []string{"init"},
 		Env: []corev1.EnvVar{
-			{Name: config.InjectionFailurePolicyEnv, Value: kubeobjects.GetField(pod.Annotations, dtwebhook.AnnotationFailurePolicy, "silent")},
```

Comment:
Changing that, we would ignore the pod.Annotations which are already present in customer environments, thats not what we should do, imho we should have an order, 1. if pod has annotation, take it, 2. if dynakube has annotation take that, 3. default (silent)

Reply:
Please check that comment :)

Reply:
`kubeobjects.GetField(pod.Annotations, dtwebhook.AnnotationFailurePolicy, FeatureInjectionFailurePolicy())}`

Reply:
have a look at my way to implement it, as i think the default value of `kubeobjects.GetField` should be "silent" and we should only check whether `FeatureInjectionFailurePolicy() `is set explicitly to "fail". if not, we just take "silent".